coalib/bearlib/languages/LanguageDefinition.py

```diff
@@ -7,7 +7,7 @@
 
 class LanguageDefinition(SectionCreatable):
 
-    def __init__(self, language: str):
+    def __init__(self, language: str, coalang_path=None):
```

Comment:
@abhsag24 alright this doesn't seem to be ready after all, wanna cherry pick out this commit? Then I'll remove it from this PR.